core/src/test/scala/akka/persistence/jdbc/journal/JdbcJournalPerfSpec.scala

```diff
@@ -104,15 +104,15 @@ abstract class JdbcJournalPerfSpec(config: Config, schemaType: SchemaType)
   }
 }
 
-class PostgresJournalPerfSpec extends JdbcJournalPerfSpec(ConfigFactory.load("postgres-application.conf"), Postgres()) {
+class PostgresPartitionedJournalPerfSpec extends JdbcJournalPerfSpec(ConfigFactory.load("postgres-application.conf"), PostgresPartitioned()) {
   override def eventsCount: Int = 100
 }
 
-class PostgresJournalPerfSpecSharedDb
-    extends JdbcJournalPerfSpec(ConfigFactory.load("postgres-shared-db-application.conf"), Postgres()) {
+class PostgresPartitionedJournalPerfSpecSharedDb
+    extends JdbcJournalPerfSpec(ConfigFactory.load("postgres-shared-db-application.conf"), PostgresPartitioned()) {
```

Comment:
#14 introduced a feature flag - you will need to provide a config with `jdbc-journal.partitioned` set to `true`.
Otherwise JournalDao might not even try to create missing partitions.

Reply:
done